configFile_LNGS.txt

```diff
@@ -19,7 +19,7 @@
 'flag_dbscan_seeds'     : 1,
 'flag_stats'            : 0,
 
-'camera_mode'            : 1,
+'camera_mode'            : 0,
```

Comment:
Shouldn't 'camera_mode' stays On?

Reply:
Yes, I'll set it to 1. It was set to 0 because I was running some tests running only the pmt reco, without the camera reco.